plugins/com.yakindu.solidity.ide/src/com/yakindu/solidity/ide/SolidityIdeSetup.xtend

```diff
@@ -28,6 +25,6 @@ import org.eclipse.xtext.util.Modules2
 class SolidityIdeSetup extends SolidityStandaloneSetup {
 
 	override createInjector() {
-		Guice.createInjector(Modules2.mixin(new SolidityRuntimeModule, new SolidityIdeModule))
+		Guice.createInjector(Modules2.mixin(new SolidityRuntimeModule, new com.yakindu.solidity.ide.internal.SolidityIdeModule))

```

Comment:
Is there a reason for using the fqn instead of import?

Reply:
No, this can acually be imported

Reply:
This was done by the Organize Imports action, because there are 2 versions of SolidityIdeModule.
com.yakindu.solidity.ide.internal.SolidityIdeModule
com.yakindu.solidity.ide.SolidityIdeModule

Reply:
updated; added the import